whois-api/src/test/java/net/ripe/db/whois/api/rdap/WhoisRdapServiceTestIntegration.java

```diff
@@ -237,7 +237,8 @@ public void lookup_inetnum_range_reserved() {
 
         assertThat(ip.getPort43(), is("whois.ripe.net"));
         assertThat(ip.getRdapConformance(), hasSize(2));
-        assertThat(ip.getRdapConformance(), containsInAnyOrder("rdap_level_0", "cidr0"));
+        assertThat(ip.getRdapConformance(), containsInAnyOrder(RdapConformanceEnum.LEVEL_0.name(),
```

Comment:
I suggest comparing with an expected **string** value, not the enum name itself (i.e. what will the string look like? Is it uppercase?)

Reply:
Just tested that "CIRD_0" appears in the response rather than the expected "cidr_0"

Reply:
yes, I am taking the name() instead of the value. I fixed